src/modules/tools/temperaturecontrol/TemperatureControl.h

```diff
@@ -83,14 +81,17 @@ class TemperatureControl : public Module {
         float PIDdt;
 
         // Temperature runaway values
+        enum RUNAWAY_TYPE {NOT_HEATING, HEATING_UP, COOLING_DOWN, MAINTAINING_TEMPERATURE};
+
         RUNAWAY_TYPE runaway_state;      
+        uint8_t runaway_heating_timer;
+        float runaway_last_temperature;
+        // Temperature runaway config options
+        uint8_t runaway_range;
+        uint8_t runaway_heating_timeout;
 
 
         struct {
-            uint8_t runaway_heating_timer:8;
```

Comment:
How would it increase memory?  It's an 8-bit integer.   On its own (as now), it would consume 8-bits of memory.  Packed into a struct it would consume 8-bits of memory.  Unless I'm missing something?